### PR TITLE
Binary Export/Import Fix & Extension

### PR DESCRIPTION
**Pull request by wahln**
This pull request should hopefully fix the issues raised in #281 and #282 (thanks).

The GUI has also been extended to handle HU cubes. It can now be selected if the cube should be converted with a hlut (and enables selection of the file).
Also it addas a test script to write and read a ct cube and check for consistency.

### CODE REVIEW

File: IO/matRad_readHLUT.m [hunk 1]
```diff
@@ -0,0 +1,52 @@
+function hlut = matRad_readHLUT(filename)
```
Comment (markbangert) [inline IO/matRad_readHLUT.m]: cant you reuse the equivalent function from the dicom import? I have seen that there also the particle type is required from the plan but maybe there is also a way around?

Comment (wahln) [reply IO/matRad_readHLUT.m]: I don't like the matRad_loadHLUT function in this context. Actually I see it more as a function to automatically determine which file to load, and thus I wanted to write a function that does nothing more than actually read the file (which is by the way now also used in matRad_loadHLUT).

**Comment by markbangert** [comment]
How would I best review and test this?

Export an entire patient (e.g. TG119) with ct, segmentations and dose cube and then import it again? Is there any other information that I should look at? Your test does not include the dose, right?

**Comment by wahln** [comment]
Yes, try to do a full export and reimport.

The test script is more suited for checking if the actual export and import routines workin general so far, so I just chose the ct to try it out.

**Pull request merged**